src/components/Footer/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './Footer.css';
+import liLogo from '../../Images/LI-In-Bug.png';
+import ghLogo from '../../Images/GitHub-Mark-Light-64px.png';
+
+const Footer = () => {
+  return (
+    <footer className='footer'>
+      <div className='dev-links'>
+        <h2>Christina Cook</h2>
+        <a href="https://github.com/christina-cook">
+          <img className="ghLogo" src={ghLogo} />
+        </a>
+        <a href="https://www.linkedin.com/in/cookcmc/">
+          <img className="liLogo" src={liLogo}/>
+        </a>
+      </div>
+      <div className='dev-links'>
+        <h2>Allison Dietz</h2>
+        <a href="https://github.com/dietza">
+          <img className="ghLogo" src={ghLogo} />
+        </a>
+        <a href="https://www.linkedin.com/in/allison-dietz/">
+          <img className="liLogo" src={liLogo}/>
+        </a>
+      </div>
+      <div className='dev-links'>
+        <h2>Cole Fiscus</h2>
+        <a href="https://github.com/colefiscus">
+          <img className="ghLogo" src={ghLogo} />
+        </a>
+        <a href="https://www.linkedin.com/in/colefiscus/">
+          <img className="liLogo" src={liLogo}/>
+        </a>
+      </div>
+    </footer>
+  )
+}
+
```

Comment:
The footer looks good to me and all links are working.

Reply:
Agreed, great start!